app/src/main/res/layout/list_item_search_overdue_select_all_button.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.button.MaterialButton xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:id="@+id/search_overdue_select_all_button"
+  style="?attr/borderlessButtonDenseStyle"
+  android:layout_width="wrap_content"
+  android:layout_height="wrap_content"
+  android:layout_marginStart="@dimen/spacing_8"
+  android:layout_marginTop="@dimen/spacing_16"
```

Comment:
The button looks like it's mis aligned. Will send screenshots in DM. The top and bottom margins are higher than in the layout and the button's body is not aligned with the card below. Can you ptal at that?